practice/blackjack/card.py

```diff
@@ -0,0 +1,27 @@
+class Card:
+    """This class holds card values"""
```

Comment:
A good thing to describe in the class docstring is what the fields represent as if they were input arguments. Talk about what suit and rank represent and what types they should be.